web/src/components/dashboard/OpportunityCard.tsx

```diff
@@ -28,9 +29,19 @@ export const OpportunityCard = (props: Props): ReactElement => {
   const router = useRouter();
 
   const TYPE_TO_LABEL: Record<"funding" | "certification", ReactElement> = {
-    funding: <Tag backgroundColor="accent-semi-cool-light">{Config.dashboardDefaults.fundingTagText}</Tag>,
+    funding: (
+      <Tag
+        isLowerCase={true}
+        className={"border-accent-semi-cool-500 border"}
+        backgroundColor="accent-semi-cool-light"
+      >
+        {capitalizeEachWord(Config.dashboardDefaults.fundingTagText)}
+      </Tag>
+    ),
     certification: (
-      <Tag backgroundColor="accent-cool-light">{Config.dashboardDefaults.certificationTagText}</Tag>
+      <Tag isLowerCase={true} className={"border-accent-cool border"} backgroundColor="accent-cool-lighter">
+        {capitalizeEachWord(Config.dashboardDefaults.certificationTagText)}
+      </Tag>
```

Comment:
It looks like we're only using `capitalizeEachWord` for 2 words, `Funding` and `Certification`. It looks like these are already capitalized. Is this being added as a safeguard to ensure it stays capitalized?